install/kubernetes/helm/Istio/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: Istio
+version: 0.1.0-1
+description: Helm chart for complete Istio deployment
+keywords:
+  - Istio
+  - auth
```

Comment:
Rename to security

Reply:
Actually forget this. We can clean up with the right keywords later (telemetry & tracing, traffic routing, security, l7 policy enforcement)

Reply:
noted